meshroom/ui/reconstruction.py

```diff
@@ -39,134 +39,6 @@ def __init__(self, title, text, detailedText="", parent=None):
-
-    File watching is based on regular polling and not filesystem events to work on network mounts.
-    """
-    def __init__(self, reconstruction):
-        super().__init__(reconstruction)
-        self.reconstruction = reconstruction
-        self._folder = ''
-        self.timerId = -1
-        self.minImagesPerStep = 4
-        self.watchTimerInterval = 1000
-        self.allImages = []
-        self.cameraInit = None
-        self.sfm = None
-        self._running = False
-
-    def reset(self):
-        self.stop(False)
-        self.sfm = None
-        self.cameraInit = None
-
-    def setRunning(self, value):
-        if self._running == value:
-            return
-        if self._running:
-            self.killTimer(self.timerId)
-        else:
-            self.timerId = self.startTimer(self.watchTimerInterval)
-        self._running = value
-        self.runningChanged.emit()
-
-    @Slot(str, int)
-    def start(self, folder, minImagesPerStep):
-        """
-        Start live SfM augmentation.
-
-        Args:
-            folder (str): the folder to watch in which images are added over time
-            minImagesPerStep (int): minimum number of images in an augmentation step
-        """
-        # print('[LiveSfmManager] Watching {} for images'.format(folder))
-        if not os.path.isdir(folder):
-            raise RuntimeError(f"Invalid folder provided: {folder}")
-        self._folder = folder
-        self.folderChanged.emit()
-        self.cameraInit = self.sfm = None
-        self.allImages = self.reconstruction.allImagePaths()
-        self.minImagesPerStep = minImagesPerStep
-        self.setRunning(True)
-        self.update()  # trigger initial update
-
-    @Slot()
-    def stop(self, requestCompute=True):
-        """ Stop the live SfM reconstruction.
-
-        Request the computation of the last augmentation step if any.
-        """
-        self.setRunning(False)
-        if requestCompute:
-            self.computeStep()
-
-    def timerEvent(self, evt):
-        self.update()
-
-    def update(self):
-        """
-        Look for new images in the watched folder and create SfM augmentation step (or modify existing one)
-        to include those images to the reconstruction.
-        """
-        # Get all new images in the watched folder
-        imagesInFolder = multiview.findFilesByTypeInFolder(self._folder).images
-        newImages = set(imagesInFolder).difference(self.allImages)
-        for imagePath in newImages:
-            # print('[LiveSfmManager] New image file : {}'.format(imagePath))
-            if not self.cameraInit:
-                # Start graph modification: until 'computeAugmentation' is called, every commands
-                # used will be part of this macro
-                self.reconstruction.beginModification("SfM Augmentation")
-                # Add SfM augmentation step in the graph
-                self.cameraInit, self.sfm = self.reconstruction.addSfmAugmentation()
```

Comment:
Need to remove code that is only used here `addSfmAugmentation` and related code like `multiview.sfmAugmentation`.

Reply:
Done, the "Augment Reconstruction" feature has been fully removed as well.